test/test.go

```diff
@@ -414,6 +417,47 @@ func (h *Helper) TempFile(path, contents string) {
 	h.Must(ioutil.WriteFile(filepath.Join(h.tempdir, path), bytes, 0644))
 }
 
+// GetTestFileBytes reads a file form the testdata directory into memory.  src is
+// relative to ./testdata.  Assumes tests take place starting in the cmd/dep
+// directory.
+func (h *Helper) GetTestFileBytes(src string) []byte {
+	content, err := ioutil.ReadFile(filepath.Join(h.origWd, "_testdata", src))
+	if err != nil {
+		panic(err)
+	}
+	if strings.HasSuffix(src, ".go") {
+		formatted, err := format.Source(content)
```

Comment:
Agreed.  Was copying the existing pattern, but I did add a "style-independent" JSON comparison to test.  If we agree that formatting like this makes the test more brittle, I am happy to remove it - just trying to go with the flow, as it were.